bundle/language_it.properties

```diff
@@ -1175,7 +1175,7 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = I genitori sono insieme
 angal.patient.patientbrowser.title                                                                     = Elenco Pazienti
 angal.patient.patientphoto                                                                             = Foto
-angal.patient.patientselection.title                                                                   = Selezione Paziente
+angal.patient.patientselection.title                                                                   = Non � stata trovata la foto del paziente.
```

Comment:
There's no need to add new keys in the language other than "en". Please revert this change.